libdrmutils/Android.mk

```diff
@@ -5,7 +5,9 @@ LOCAL_MODULE                  := libdrmutils
 LOCAL_VENDOR_MODULE           := true
 LOCAL_MODULE_TAGS             := optional
 LOCAL_C_INCLUDES              := external/libdrm \
```

Comment:
LOCAL_C_INCLUDES              := external/libdrm
not
LOCAL_C_INCLUDES              := external/libdrm \